packages/bot-web-ui/src/pages/dashboard/announcements/announcements.scss

```diff
@@ -0,0 +1,209 @@
+@import '../../../styles/mixins.scss';
+
+.announcements {
+    position: absolute;
+    inset-inline-end: 2.4rem;
+    top: 1.8rem;
+    z-index: 1;
+
+    @include mobile-or-tablet-screen {
+        inset-inline-end: 1.6rem;
+    }
+
+    &__button {
+        @include flex-center;
+        padding: 0.8rem 1.4rem;
+        border: none;
+        outline: none;
+        border-radius: 2.4rem;
+        background-color: var(--general-section-1);
+        cursor: pointer;
+
+        @include mobile-or-tablet-screen {
+            height: 3.2rem;
+            width: 3.2rem;
+            line-height: 3.6rem;
+            text-align: center;
+            padding: 0;
+        }
+    }
+
+    &__icon {
+        height: 2.4rem;
+        width: 2.4rem;
+
+        @include mobile-or-tablet-screen {
+            height: 2.6rem;
+            width: 2.6rem;
+        }
+    }
+
+    &__label {
+        margin-inline-start: 0.4rem;
+    }
+
+    &__amount {
+        background-color: var(--status-danger);
+        border-radius: 0.9rem;
+        height: 1.8rem;
+        width: 1.8rem;
+        margin-inline-start: 0.4rem;
+        color: var(--brand-white);
+        font-weight: bold;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        @include mobile-or-tablet-screen {
+            position: absolute;
+            bottom: 2.1rem;
+            left: 1.8rem;
+        }
+    }
+}
+
+.notifications__wrapper {
+    padding-inline-start: 0.5rem;
+    border-inline-start: 1px solid var(--general-section-1);
+    border-width: 1px;
+    margin-inline-end: 1rem;
+
+    &--desktop {
+        z-index: 10;
+        top: 5rem;
+        right: 0;
+        position: absolute;
+        width: 40rem;
+
+        @include rtl {
+            right: -24rem;
+        }
+    }
+
+    .deriv-text {
+        font-weight: bold;
+    }
+
+    .deriv-context-menu {
+        box-shadow: 0 3.2rem 6.4rem 0 #0e0e0e24;
+    }
+}
+
+.notifications {
+    min-height: 53.6rem !important;
+    background-color: var(--general-main-2);
+
+    @include mobile-or-tablet-screen {
+        height: 100svh;
```

Comment:
@maryia-matskevich-deriv can you explain why are we using svh ?

Reply:
@rupato-deriv The **svh** unit stands for Small Viewport Height and represents 1% of the smallest viewport height available across all viewports. It’s a relatively new CSS unit introduced to better handle viewport size inconsistencies across different devices and browser interfaces.
In this context, 100svh means the element’s height will always cover 100% of the smallest viewport height, reducing unexpected jumps or content getting cut off.
Using **svh** is beneficial when you want to stabilize the height of elements across varying viewport conditions, especially on mobile devices where the viewport can change dynamically as interface elements come in and out of view.

In our case, we face the situation when our modal window perfectly fits on the mobile device screen in Chrome, but once we check this situation on the real device it looks cropped, property ` height: 100vh;` doesn't work as we expect it, for example on real device iPhone 14.

Reply:
![image](https://github.com/user-attachments/assets/f6c16585-0311-4b37-a82a-44829259b6cd)
@maryia-matskevich-deriv the `svh`  does not have support on all the browsers 

also we have used inner height in the `app.tsx`.
<img width="865" alt="image" src="https://github.com/user-attachments/assets/1c281f86-47c2-4a7a-b900-19a0a1450624">

can we try to use this ? and adjust the height without svh.

Reply:
I have tried using svh, dvh earlier in deriv-app. but the eslint gives me an error. Now sure how @maryia-matskevich-deriv implemented it.

Reply:
@sandeep-deriv and I decide to keep using svh. Rupato thank you for raising this🤗